tools/target.py

```diff
@@ -15,18 +15,35 @@ def register(command):
         "target", nargs="?", choices=platform.list(), help="the platform to select as the default platform"
     )
 
+    command.add_argument(
+        "-l",
+        "--local-build",
+        dest="build_local",
+        action="store_true",
+        default=False,
+        help="build the image locally, instead of pulling and tagging it",
```

Comment:
`./b format` adds the comma at the end of this line. Weird, but sure lol...

Reply:
Of course, trailing commas are the correct way to do multi line lists.
The rationale is that if you were to add or remove an item from the list the diff would only be for that line. If you didn't have the trailing comma then when you add a new element it would be a diff of adding a comma to the previous (unrelated) line as well as adding the new line.

Reply:
Yep, trailing commas all the way